### PR TITLE
fluff: make rollback links and separators selectable via CSS

### LINKED ISSUE #1386: Make [vandalism] rollback links properly hideable via CSS
Instead of an external stylesheet, Twinkle uses inline CSS for the rollback links. This can be fine, but leads to issues when attempting to hide specific elements using CSS selectors in one's Special:MyPage/common.css.

A practical example that affects me: I currently (attempt to) use the following code to hide the red `[vandalism]` rollback link in favor of the MediaWiki core rollback link (which has a convenient JavaScript-based confirmation feature and is still faster at pushing the change to the database on large pages).

My custom CSS:

```
/* Hide Twinkle's "rollback" and "vandalism" links in favor of core rollback link in Special:Contributions */
.mw-special-Contributions .mw-contributions-current strong>a {
    display: none;
}

/* Hide Twinkle's (non-warning, instant-revert, default-summary) "vandalism" link in favor of the core rollback link above diffs */
.diff #tw-revert strong:nth-child(3) {
	display: none;
}
```

These CSS selectors are pretty far-fetched. MediaWiki is full of classes and IDs, Twinkle lacks them. However, I'd even use my code if it worked. It doesn't:

twinkle-selectors-issue-20210417

The very least I need is a workaround to remove the two pipes between the blue "rollback" and the red "vandalism" link above diffs.

### PR DESCRIPTION
**Pull request by ToBeFree**
fix #1386

twinklefluff-20210425-span
twinklefluff-20210425-sep
twinklefluff-20210425-a

### CODE REVIEW

File: modules/twinklefluff.js [hunk 1]
```diff
@@ -118,10 +129,14 @@ Twinkle.fluff.linkBuilder = {
...
 
-		var separator = inline ? ' ' : ' || ';
+		normNode.appendChild(sepNode1);
+		vandNode.appendChild(sepNode2);
+
+		normNode.appendChild(normLink);
+		vandNode.appendChild(vandLink);
 
 		if (!inline) {
 			var agfNode = document.createElement('strong');
```
Comment (Xi-Plus) [inline modules/twinklefluff.js:142]: You have changed norm and vand buttons from `<strong>` to `<span>`, but you forget it for agf button.

Comment (ToBeFree) [reply modules/twinklefluff.js:142]: That's correct, sorry. Thanks for catching this.

Comment (ToBeFree) [reply modules/twinklefluff.js]: Fixed in 9992f5d1a10835b636a853bd5692b94f713a72f5:

twinklefluff-20210430-patchfix1

twinklefluff-20210430-patchfix2

twinklefluff-20210430-patchfix3

**Comment by ToBeFree** [comment]
@Xi-Plus thanks for the review, I think I have now fixed the issue. The "request re-review" button doesn't seem to work for me :)

**Comment by siddharthvp** [comment]
@ToBeFree Sorry for the long delay, and thanks for the patch. Is it intentional that `.tw-rollback-link-separator` is inside `.tw-rollback-link-normal` or `. tw-rollback-link-vandalism`. This makes it possible to hide the normal and vandalism links, but not the AGF link.

**Comment by ToBeFree** [comment]
No worries, thank you very much for reviewing. :) That was in response to https://github.com/wikimedia-gadgets/twinkle/issues/1386#issuecomment-822183967 ; I'll have a look again but think it should be fine

**Comment by ToBeFree** [comment]
I can't see a different way to implement this, and it would work for the use-case described in #1386 that led to the whole discussion.

**Comment by ToBeFree** [comment]
@siddharthvp, hiding the AGF link can be done by targeting both the AGF link and the normal rollback's separator. Should be fine.

**Pull request merged**